cmd/bnsd/app/datamigration.go

```diff
@@ -0,0 +1,43 @@
+package bnsd
+
+import (
+	"context"
+
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/datamigration"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/gconf"
+	"github.com/iov-one/weave/x/msgfee"
+)
+
+func init() {
+	admin, err := weave.ParseAddress("cond:gov/rule/0000000000000003")
```

Comment:
@davepuchyr can you confirm that this should be the admin of the `msgfee` extension? Admin can modify configuration further and set for example who is allowed to change prices.

Reply:
Confirmed.